### ISSUE TITLE
[Feature]: Videos under artists profile

### ISSUE THREAD

**Issue by Cszyn** [fast4x/RiMusic#1966]
### Description

TL;DR This is a proposal to show videos in artist pages under a new tab but excluded from overview.

This will also affect #1812. I'd like to re propose #1830 but more though out and less invasive to the regular songs which should remain the priority as always

RIs rather hardened stance against video playback has been softening as of late and with the introduction of the ability to search for videos under a separate tab I think a similar concept should be implemented as well for the artist page.

### Suggested Solution

There is more than enough space for a new page with the side menu design, as for Top and Bottom the icons could be scaled back a little and for overflowed words being converted to a traditional “…” cut off or scrolling text could solve that.

![MuMuPlayer_SDFeQ6P5Up](https://github.com/fast4x/RiMusic/assets/147027531/cc837533-48a0-4a64-a3e0-2c8d7af48305)

![MuMuPlayer_1Lx4qDXIL4](https://github.com/fast4x/RiMusic/assets/147027531/90cde9bb-5f7f-4446-947c-30f27ec08bdf)

![MuMuPlayer_JV3zI54dps](https://github.com/fast4x/RiMusic/assets/147027531/ca812042-1d65-4873-8ffa-e3ba46fab15b)

### Alternatives

Discussion welcome
### Additional Context

Low priory as it is rater controversial

**Comment by Cszyn** [comment]
> That's a great idea, dude That's what I've been missing It's in the search, but the artists don't have it Um ...
> 
> Clips ...

? Clips?
